### PR TITLE
feat: Feature/dino batching

### PR DESCRIPTION
**Pull request by shondle**
Adds ability to batch inputs to DINO backend to retrieve bounding box or mask predictions from SAM by using the task view menu to select tasks and retrieve predictions given user prompt input. 

https://github.com/HumanSignal/label-studio-ml-backend/assets/106922533/b943acc4-3ad4-44a7-9b79-41180bc59c79

### CODE REVIEW

File: label_studio_ml/examples/grounding_dino/dino.py [hunk 1]
```diff
@@ -103,35 +179,140 @@ def predict(self, tasks: List[Dict], context: Optional[Dict] = None, **kwargs) -
...
+                    all_scores.append(logit)
+                    all_lengths.append((H, W)) # figure out how to get this
+                
+                predictions.append(self.get_results(all_points, all_scores, all_lengths))            
+
+        return predictions
+            
+    # make sure you use new github repo when predicting in batch
+    def batch_dino(self, image_paths):
+        print("0here1")
```
Comment (bmartel) [inline label_studio_ml/examples/grounding_dino/dino.py:220]:
```suggestion
```

File: label_studio_ml/examples/grounding_dino/dino.py [hunk 2]
```diff
@@ -103,35 +179,140 @@ def predict(self, tasks: List[Dict], context: Optional[Dict] = None, **kwargs) -
...
 
-            boxes, logits, _ = predict(
-                model=groundingdino_model,
-                image=img,
-                caption=TEXT_PROMPT.strip("_SAM"),
-                box_threshold=float(BOX_TRESHOLD),
-                text_threshold=float(TEXT_TRESHOLD),
-                device=DEVICE
-            )
+        print(image_paths)
```
Comment (bmartel) [inline label_studio_ml/examples/grounding_dino/dino.py:184]:
```suggestion
```

**Review by hogepodge — APPROVED** [review]

**Comment by NicholasFlamy** [comment]
Y'all forgot to update the readme.
